kong/clustering/version_negotiation/services_requested.lua

```diff
@@ -0,0 +1,6 @@
+return {
```

Comment:
This file is per endpoint, right? Or it is fine to list all the requested_services here, supposing we have two different endpoints that support different services?

Reply:
there's no "endpoint" concept in this code, in fact not even a hard definition of what a "service" actually is.  this negotiation has it's own endpoint and doesn't care what the negotiated things are.   This file holds everything the DP will request from the CP, in a single request/response.